corehq/apps/locations/permissions.py

```diff
@@ -99,7 +99,7 @@
 
 LOCATION_SAFE_TASTYPIE_RESOURCES = set()
 
-LOCATION_SAFE_HQ_REPORTS = set()
+LOCATION_SAFE_HQ_REPORTS = set(['case_list'])
```

Comment:
I just tried it out locally, you can just add `@location_safe` to the class right at the top: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/standard/cases/basic.py#L185